creation/cloud-init.md

```diff
@@ -36,11 +36,22 @@ detect the ephemeral disk and execute custom userdata scripts at boot.
 
 Sysprep is an automation tool for Windows that automates Windows
 installation, setup, and custom software provisioning.
+The general flow is to seal the vm image with the Sysprep tool,
+for example by running:
+```
+%WINDIR%\system32\sysprep\sysprep.exe /generalize /shutdown /oobe /mode:vm
+```
+More information can be found here:
+https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep--generalize--a-windows-installation
+https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/sysprep-process-overview
+
+And then providing an Answer file named `autounattend.xml` in an attached media.
+The answer file can be provided in a ConfigMap or a Secret with the key `autounattend.xml`
 
-**Sysprep support is currently not implemented by KubeVirt.** However it
-is a feature the KubeVirt upstream community has shown interest in. As a
-result, it is likely Sysprep support will make its way into a future
-KubeVirt release.
+More information can be found here:
+https://docs.microsoft.com/en-us/windows-hardware/manufacture/desktop/update-windows-settings-and-scripts-create-your-own-answer-file-sxs
```

Comment:
What about adding that link to the other two a couple of lines above (44-46)?

While doing this it might be worth making that a list, and/or making the links proper hyperlinks with `[title](url)`

Reply:
This section needs improvement.  It seems that two points are trying to be made here and they are blending together.  The bullet pointed list doesn't render correctly. 

See screenshots

Reply:
<img width="1060" alt="GitHub" src="https://user-images.githubusercontent.com/2479463/108103929-7863f780-703f-11eb-83f8-3f1cbf8af1b0.png">

Reply:
<img width="1139" alt="Netlify" src="https://user-images.githubusercontent.com/2479463/108104115-bcef9300-703f-11eb-85a1-0e0c585b7163.png">